### ISSUE TITLE
Ask for the training curves

### ISSUE THREAD

**Issue by sangwoomo** [OctThe16th/FIGR#4]
Hi, I tried to reproduce your results on MNIST dataset, but the model seems to diverge.
I simply clone your repo and run `python train.py --dataset Mnist`.

Here are my loss curves and generated samples.
![loss](https://user-images.githubusercontent.com/16223881/59677364-9ba02900-9204-11e9-8eb6-c54948dc8c48.png)
![sample](https://user-images.githubusercontent.com/16223881/59677393-abb80880-9204-11e9-82ea-9521d7c9078e.png)

Can you share your loss curves and generated samples during training?
(I'm using PyTorch v1.1)

**Comment by sangwoomo** [comment]
It seems to work well after thousands of epochs.
![sample](https://user-images.githubusercontent.com/16223881/59683048-5daa0180-9212-11e9-9ad2-41a5f8b5b7d6.png)

**Comment by LuEE-C** [comment]
The training curves just mean that the discriminator is getting better and better in relation to the generator's performance and do not necessarily imply divergence in a gan. I do not have access to training curves right now but this seems right to me. Do note that it takes 10's of thousands of steps to produce good results
